generated/visa/visa.proto

```diff
@@ -496,13 +495,21 @@ message ClearResponse {
 }
 
 message CloseRequest {
-  SessionOrEventData object_handle = 1;
+  nidevice_grpc.Session vi = 1;
 }
 
 message CloseResponse {
   int32 status = 1;
 }
 
+message CloseEventRequest {
+  uint32 event_handle = 1;
+}
```

Comment:
This event handle is generated on the server by the call to WaitForEvent, right?

Consider allowing the client to assign a stable name for the event. See @ccifra's reasoning in  https://github.com/ni/grpc-device/pull/784:

> I don't think ID should ever be used. There are a lot of advantages to keeping the data on the wire "time invariant" or independent on server state. For example, we are trying to do batching to improve throughput when doing "cloud-based measurements" as part of various LTIs and hopefully products.

Reply:
It looks like job identifiers for async calls are also server-generated tokens.

Reply:
Async jobs and events are so temporal, I would argue strongly against naming them. There's really no way that ever makes sense to me. 

I agree completely that long-lived objects such as sessions should always be named.

Reply:
I should have said "stable identifier". It doesn't have to be a name. Allowing the client to choose a numeric identifier would enable batching of multiple calls, record/replay use cases, etc.

Would it be possible to implement this later? Currently, the server has a single flat namespace for event handles, so clients would not be able to choose their own event handles without causing collisions. If event handles were scoped by session, this would be less of a problem (assuming the session is accessed by only one client at a time, and each client completes any events or jobs before another client takes ownership of the session).

Reply:
After the weekend, I still don't think it makes sense for the 99% case. Consider LV code. The user never gets access to the job ID or the event ID. A single LV read/write operation encapsulates all of [begin, wait, get completion data from the event, close the event]. Even service requests are hidden behind a VI that does [wait, close the event].

I would hope this would be something that one could add later if there were a useful business case, but I still can't grasp that in my mind at this time.

Reply:
Ok, thank you for considering it.